R/model_identify_variables.R

```diff
@@ -71,7 +71,7 @@ model_identify_variables.default <- function(model) {
     ) %>%
     dplyr::left_join(
       model_list_variables(model) %>%
-        dplyr::select(.data$variable, .data$var_class),
+        dplyr::select(dplyr::all_of(c("variable", "var_class"))),
```

Comment:
This could just be `select("variable", "var_class")`

Reply:
@hadley thank you for taking a look at these changes. so kind!

🧐  thank you for the clarification...i've been putting all character vectors in `add_of()`/`any_of()` since they were released!